### DISCUSSION TITLE
Authorization options lock me out of a test restore.

### DISCUSSION THREAD

**Discussion by NKNdevs** [outline/outline#7981] [Self Hosting]
I decided to try a test restore of our Outline installation as we've decided to go ahead and use it, after a period of testing. 
I span up a new server on test.domain.com, installed docker, got the backups onto the new server, and, I think, successfully restored them. 
However I'm unable to verify that as we have Google auth enabled. When I try to login with that, I get sent to the current live server outline.live.com, as that's where the callback URL sends us. 

What I'm looking for is a way to login to the  restored instance on test.domain.com so I can verify the data. Magic Email signup seems like a good way to test it. HOwever, I can't see how to enable that. I tried disabling google auth in the docker.env file, and then had no way to login. 

Any suggestions on how to proceed. I've been looking around for a couple of hours and haven't turned up anything. If there is no way of doing this, then it might be a good feature to include. A couple of lines in docker.env like this would be useful.
ENABLE_TEST_MODE=yes
TEST_MODE_USER=bob
TEST_MODE_PASS=1234

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-757/authorization-options-lock-me-out-of-a-test-restore">OLN-757 Authorization options lock me out of a test restore.</a></p>
